examples/interfaces.mirah

```diff
@@ -17,7 +17,7 @@ import java.util.List
 
 interface Printer do
   def printAll(a:List)
-    returns void
+    return void
```

Comment:
Neat. I didn't know that would work. Now, you can just say `def printAll(a: List): void; end`, which is probably better.